LDAR_Sim/src/programs/method.py

```diff
@@ -94,7 +94,7 @@ def initialize_crews(self, crews, sites: "list[Site]") -> None:
         return
 
     def initialize_cost_tracking(self, cost_properties: dict[str, float]):
-        self.upfront_cost = cost_properties[pdc.Method_Params.UPFRONT]
+        self.upfront_cost = cost_properties[pdc.Method_Params.UPFRONT] * self._crews
```

Comment:
Just to verify - is this how we say it will work in the documentation

Reply:
Updated the doc.